### ISSUE TITLE
Columns with same name (on different tables) are messing up the results

### ISSUE THREAD

**Issue by joaofraga** [EverythingMe/redash#571]
If we have two columns with the same name in the same query the result is overridden. In this query I also found another bug: the `COUNT` and the `GROUP_CONCAT` does not show results if they don't have an alias.

```sql
SELECT s.name,
       t.name,
       count(tg.id),
       group_concat(tg.taggable_id
                    ORDER BY tg.taggable_id ASC separator ', ')
FROM taggings tg,
     tags t,
     questions q,
     subgroups s
WHERE t.id = tg.tag_id
  AND tg.taggable_id = q.id
  AND tg.taggable_type = 'Question'
  AND q.year >= 2009
  AND s.id = q.subgroup_id
GROUP BY s.name,
         t.id
```

Result:
![screen shot 2015-09-22 at 2 07 53 pm](https://cloud.githubusercontent.com/assets/501867/10025613/803d60b6-6133-11e5-8dc4-2c5f202b9f6c.png)

Should be:
![screen shot 2015-09-22 at 2 08 52 pm](https://cloud.githubusercontent.com/assets/501867/10025622/829eaf5e-6133-11e5-8daa-751fc53162a6.png)

Adding an alias to all columns fixes it.

**Comment by arikfr** [comment]
I've implemented a [solution](https://github.com/EverythingMe/redash/blob/master/redash/query_runner/pg.py#L134-L136) for this in the PostgreSQL query runner, but never generalized it for the other query runners.

Which one are you using?

**Comment by joaofraga** [comment]
@arikfr  I'm using mysql. Should I implement the solution for mysql or try to generalize it?

**Comment by arikfr** [comment]
I missed your comment about the issue with count/group_concat when I first read this issue. The problem isn't with count/group_concat themselves, but with the dot in the resulting column names. I fixed this in #577.

**Comment by joaofraga** [comment]
Great! Thank you @arikfr
